tailwind.config.js

```diff
@@ -1,29 +1,59 @@
+const colors = require('tailwindcss/colors');
+
 module.exports = {
   mode: 'jit',
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false, // or 'media' or 'class'
   theme: {
+    colors: {
```

Comment:
📓 (non-blocking): we should leave a comment/document -- notes that one should delete colors not being used to reduce css-bundle size.